bstoudt01/BrewMore
# Edit brand

# Description 
Edit brand is now populating with data and updating correctly. But the delete of the join table elements is not working properly, the DOM and state to do refresh correctly but the post, put, and deletes all work.

## Type of chang: 
Added Edit Functionality to Brand Cards

# Testing Instructions
Launch and login to set credentials, check brand list, select a brand to edit.
Edit brand values, add new ingredient values, remove new ingredient useState values, delete join table ingredient values

# Checklist:
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have added test instructions that prove my fix is effective or that my feature works
